backoffice/tests/UnitTests/Helpers/ConstantsTests.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics.Contracts;
+using System.Text.RegularExpressions;
+using Droits.Helpers;
+
+namespace Droits.Tests.UnitTests.Helpers;
+
+public class ConstantsTests
+{
+    [Fact]
+    public void PostcodeMatch_WithPostcodeWithSpace_ShouldReturnTrue()
+    {
+        var postcode = "BI9 3LF";
+        var regex = new Regex(Constants.PostcodeRegex);
```

Comment:
As this same regex is used everywhere, you could move this to be global and initialised in the constructor